ui/src/templates/actions/index.ts

```diff
@@ -0,0 +1,50 @@
+import _ from 'lodash'
+
+//Types
+import {TemplateSummary, DocumentCreate} from '@influxdata/influx'
+import {RemoteDataState} from 'src/types'
+
+// API
+import {client} from 'src/utils/api'
+
+export enum ActionTypes {
+  GetTemplateSummariesForOrg = 'GET_TEMPLATE_SUMMARIES_FOR_ORG',
+  PopulateTemplateSummaries = 'POPULATE_TEMPLATE_SUMMARIES',
+  SetTemplatesStatus = 'SET_TEMPLATES_STATUS',
+}
+
+export type Actions = PopulateTemplateSummaries | SetTemplatesStatus
+
+export interface PopulateTemplateSummaries {
+  type: ActionTypes.PopulateTemplateSummaries
+  payload: {items: TemplateSummary[]}
+}
+
+export const populateTemplateSummaries = (
+  items: TemplateSummary[]
+): PopulateTemplateSummaries => ({
+  type: ActionTypes.PopulateTemplateSummaries,
+  payload: {items},
+})
+
+export interface SetTemplatesStatus {
+  type: ActionTypes.SetTemplatesStatus
+  payload: {status: RemoteDataState}
+}
+
+export const setTemplatesStatus = (
+  status: RemoteDataState
+): SetTemplatesStatus => ({
+  type: ActionTypes.SetTemplatesStatus,
+  payload: {status},
+})
+
+export const getTemplatesForOrg = (orgName: string) => async dispatch => {
+  dispatch(setTemplatesStatus(RemoteDataState.Loading))
```

Comment:
template status gets set to loading; does it need to get set to done after populating the template summaries?

Reply:
it does as part of the populateTemplateSummaries reducer, but brought it to the action to make more discoverable